Formula/opencascade.rb

```diff
@@ -0,0 +1,49 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+  head "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=HEAD;sf=tgz"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "freeimage"
+  depends_on "freetype"
+  depends_on "gl2ps"
+  depends_on "tbb"
+
+  def install
+    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    mkdir "build" do
+      system "cmake", "..", "-DUSE_GL2PS=ON",
+                            "-DUSE_FREEIMAGE=ON",
+                            "-DUSE_TBB=ON",
+                            "-DINSTALL_DOC_Overview=ON",
+                            "-D3RDPARTY_FREETYPE_DIR=#Formula['freetype'].opt_prefix",
+                            "-D3RDPARTY_FREEIMAGE_DIR=#Formula['freeimage'].opt_prefix",
+                            "-D3RDPARTY_FREEGL2PS_DIR=#Formula['gl2ps'].opt_prefix",
+                            "-D3RDPARTY_TBB_DIR=#Formula['tbb'].opt_prefix",
+                            "-D3RDPARTY_TCL_DIR:PATH=/usr",
```

Comment:
`:PATH` is redundant

Reply:
Fair-enough.  I have vi setup to highlight column 80 and will dutifully oblige it's warnings when working on Homebrew formula. 

That said, while `brew audit --ilovezfs` works great, is this by chance another `brew audit --strict` opportunity?

Reply:
If only it were always possible to stay <= 80 … The actual enforced maximum by `brew style` right now is 244 (:see_no_evil:) so you're stuck with `--ilovezfs`.